tests/test_fractal_dimension.py

```diff
@@ -0,0 +1,55 @@
+import numpy as np
+import pytest
+from hilbertcurve.hilbertcurve import HilbertCurve
+
+import cloudmetrics
+
+
+def test_hilbert_curve():
```

Comment:
So awesome you added this!

Reply:
It's nice that someone actually went through the trouble of writing a Hilbert curve package, it's super easy to use :)